### ISSUE TITLE
Widget to display no. of followers on beacon card

### ISSUE THREAD

**Issue by Omegaviv** [CCExtractor/beacon#131]
# Feature improvement
Adding a widget icon to display the number of followers on the beacon card.

Also on tap on that icon widget, we can display the names of the followers.

This can be a handy feature, where users can directly view the number of followers and can check them without opening the hike screen.

<img src="https://user-images.githubusercontent.com/86337946/153157556-f8bb6a95-72c6-4f25-b4ba-592a1734bc33.PNG" height="500">

@nb9960 is this feature seems good?

**Comment by ItsAdityaKSingh** [comment]
Maybe the cards must be left with only the useful information being shown atm. Adding more and more information could make it chaotic and less appealing. The number of followers shouldn't matter ig. Your opinion?

**Comment by Omegaviv** [comment]
Yes your are right more information could make it chaotic. But also showing followers on the card would give a feel that it is some kind of room, so the user knows clicking the card would provide more details. What do you think?

**Comment by Omegaviv** [comment]
Yes I think you have a point, leaving it simple and clean would be appropriate. Thanks✌️

**Comment by ItsAdityaKSingh** [comment]
Thanks for understanding my views @Omegaviv! I am open to any other discussion you would like to have or propose :)

**Comment by ItsAdityaKSingh** [comment]
@Omegaviv, I guess you could close the issue now as it no longer requires to be resolved.
